### PR TITLE
Video transcoding api setup

### PR DESCRIPTION
**Pull request by Vagn99**
## Description

We have changed the GCP handling routes, so they are now using the new video/transcoding service.

## Changes

All of the routes now basically just forwards requests to the video/transcoding service.

## Related Issues

NIL

## Checklist

- [x] Code has been tested locally and passes all relevant tests.
- [x] Documentation has been updated to reflect the changes, if applicable.
- [x] Code follows the established coding style and guidelines of the project.
- [x] All new and existing tests related to the changes have passed.
- [x] Any necessary dependencies or new packages have been properly documented.
- [x] Pull request title and description are clear and descriptive.
- [x] Reviewers have been assigned to the pull request.
- [x] Any potential security implications have been considered and addressed.
- [x] Performance impact of the changes has been evaluated, if relevant.

## Screenshots (if applicable)

NIL

## Required front end merge!

This will break the loading of images and videos on the front end until this commit from the front end has been merged into its dev:

Branch: video_pr_10_11

Commit id: 48301bf

## Notes for Reviewers

Please just test the routes locally in Postman :)

### CODE REVIEW

**Review by Donhusum — APPROVED** [review]

File: routes/bucketRoutesGCP.js [hunk 1]
```diff
@@ -1,216 +1,82 @@
...
-// Creates a client
-
-// New GCP Bucket Instance
-const storage = new Storage({
-  projectId: credentials.project_id,
-  keyFilename: credentials,
+const axios = require("axios");
+const FormData = require('form-data');
+
+const serviceUrl = "https://video-service-staging-x7rgvjso4a-ew.a.run.app/";
```
Comment (Havkost) [inline routes/bucketRoutesGCP.js:6]: Fjern ekstra / sidst i URL'en.
Måske den burde rykkes til en environment fil ell.lign?

**Review by Havkost — CHANGES_REQUESTED** [review]
## Changes requested
Lille ændring + overvejelse ift. senere error handling

File: routes/bucketRoutesGCP.js [hunk 2]
```diff
@@ -1,216 +1,82 @@
...
+const storage = multer.memoryStorage();
+const upload = multer({ storage: storage });
+
+// Get list of all files in bucket
+router.get("/", (req, res) => {
+  //Forward to service api
+  axios.get(serviceUrl + "/bucket/").then((response) => {
+    res.send(response.data);
+  }).catch((error) => {
+    res.send("Error: " + error);
```
[the remark this answers is not in the document]
Comment (sofiiagran) [reply routes/bucketRoutesGCP.js:17]: Skal man ikke bruge error codes ?

[the remark this answers is not in the document]
Comment (sofiiagran) [reply routes/bucketRoutesGCP.js:17]: måske lave en try catch i stedet?

File: routes/bucketRoutesGCP.js (hunk 1 above)
Comment (Vagn99) [reply routes/bucketRoutesGCP.js]: Den er rykket til .evn nu!

**Review by JackSMunn — DISMISSED** [review]

**Review by JackSMunn — APPROVED** [review]
I approve, if the changes @Havkost has requested are completed.

File: routes/bucketRoutesGCP.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Vagn99) [reply routes/bucketRoutesGCP.js]: Der er styr på errors nu 😎
image

File: routes/bucketRoutesGCP.js (hunk 1 above)
Comment (Vagn99) [reply routes/bucketRoutesGCP.js]: image

**Pull request merged**